### ISSUE TITLE
Too many constraints force composer to stay on 1.10.4 for PHP 8

### ISSUE THREAD

**Issue by jacquesbh** [Sylius/Sylius#13323]
**Sylius version affected**: 1.10.5

**Description**  

Since composer always tries to have the most up-to-date packages installed, with all the conflicts in Sylius 1.10.5 it will always go for 1.10.4 if you require `sylius/sylius=^1.10`, which is the case for Sylius-Standard.

**So right now if you install Sylius using `composer create-project` you will get a sylius 1.10.4, and therefore get the #13211 bug.**
Which is quite unconvenient…

**Steps to reproduce**  

With PHP 8 installed.

```
composer create-project --prefer-dist --no-scripts --no-dev --no-progress sylius/sylius-standard sylius "~1.10.0"
cd sylius
composer show sylius/sylius | grep versions
```
It will give 1.10.4 besides 1.10.5 is available.

To continue and check that 1.10.5 is completely compatible with PHP 8:

```
composer require sylius/sylius=1.10.5 --with-all-dependencies
composer show sylius/sylius | grep versions
```

It shows sylius 1.10.5.

**Possible Solution**  

Check all constraints and remove them or update them.

Actually this issue is something we call "Dependency Hell".

Another "solution" could be to force `^1.10.5` on `sylius/sylius-standard`'s composer.json.

**Comment by maximehuran** [comment]
We have the same issue in our plugins.
The first time, it installs 1.10.5 : 

<img width="482" alt="image" src="https://user-images.githubusercontent.com/11380627/143569178-961d451f-6e0b-407a-bd9e-7f3a4ab5919e.png">

The second time, everything is updated to 1.10.4 dependencies : 

<img width="361" alt="image" src="https://user-images.githubusercontent.com/11380627/143569238-38238885-4660-4854-8928-13cef57b01f3.png">

The third time, I have `Nothing to install` : 

<img width="461" alt="image" src="https://user-images.githubusercontent.com/11380627/143569274-d087c752-f2d7-4215-be9c-211ba737c2b2.png">

**Comment by dsbe-ak** [comment]
Facing the same issue, 1.10.5 won't install for "^1.10", only by forcing "v1.10.5" (which is not an option in my opinion).

**Comment by dsbe-ak** [comment]
Seems to be fixed with sylius version 1.10.6!

**Comment by Zales0123** [comment]
Indeed, it seems to be fixed 🎉 @jacquesbh please, reopen if it's still an issue :) 🖖
